packages/ihe-gateway-sdk/src/models/patient-discovery/patient-discovery-responses.ts

```diff
@@ -15,7 +15,7 @@ export const inboundPatientResourceSchema = z.object({
       })
     )
     .optional(),
-  gender: z.enum(["male", "female", "unknown"]).optional(),
+  gender: z.enum(["male", "female", "undifferentiated"]).optional(),
   birthDate: z.string().optional(),
```

Comment:
should we be adding `undifferentiated`, rather than replacing `unknown`?

Reply:
Undifferentiated is the correct third value in the gender field as opposed to unknown according to the IHE spec